server/src/web/user/user.controller.spec.ts

```diff
@@ -0,0 +1,150 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { UserController } from './user.controller';
+import {instance, mock, when} from 'ts-mockito';
+import {UserService} from '../../app/user/user.service';
+import {UserServiceImpl} from '../../app/user/user.service.impl';
+import {User} from '../../domain/user/user.entity';
+import {UserDto} from '../../app/user/dto/user.dto';
+import {BadRequestException} from '@nestjs/common';
+
```

Comment:
we should specify error(ex InvalidParameterException, ValidationException... ) such as BadRequestException